src/v/kafka/server/quota_manager.h

```diff
@@ -111,33 +145,16 @@ class quota_manager : public ss::peering_sharded_service<quota_manager> {
       uint32_t mutations,
       clock::time_point now);
 
-    // throttle, return <previous delay, new delay>
-    std::chrono::milliseconds throttle(
+    std::chrono::milliseconds cap_to_max_delay(
```

Comment:
I think a lot of the `duration_cast` noise can be alleviated by working in `clock::duration`

Reply:
Beautiful 🙌